src/libs/next-auth/sso-providers/google.ts

```diff
@@ -0,0 +1,23 @@
+import Google from 'next-auth/providers/google';
+
+import { authEnv } from '@/config/auth';
+
+import { CommonProviderConfig } from './sso.config';
+
+const provider = {
+  id: 'google',
+  provider: Google({
+    ...CommonProviderConfig,
+    authorization: {
+      params: {
+        scope:
+          'openid https://www.googleapis.com/auth/userinfo.profile https://www.googleapis.com/auth/userinfo.email openid',
+      },
+    },
+    // TODO(NextAuth ENVs Migration): Remove once nextauth envs migration time end
+    clientId: authEnv.AUTH_GOOGLE_CLIENT_ID ?? process.env.AUTH_GOOGLE_CLIENT_ID,
```

Comment:
just `process.env.AUTH_GOOGLE_CLIENT_ID` , no need the authEnv.AUTH_GOOGLE_CLIENT_ID